.github/workflows/main.yml

```diff
@@ -3,7 +3,7 @@ name: raven-wps Testing
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
When will this default branch name change be "live"?  You will take care to sync with the changes here https://github.com/bird-house/finch/issues/355#issuecomment-2096900447 to ensure the notebook autodeploy and nightly Jenkins is not broken?

Reply:
This has been live for Raven for some time now. I imagine that there aren't any problems.

Reply:
Ah true the branch switch has been already done for Raven, I forgot.  So this github CI workflow change was missing to complete the switch.

All good then.